blis-src/build.rs

```diff
@@ -83,6 +89,8 @@ fn main() {
             "cargo:rustc-link-search=native={}",
             lib_dir.to_string_lossy()
         );
+        let include_dir = out_dir.join("include");
+        println!("cargo:include={}", include_dir.to_string_lossy());
```

Comment:
Thanks. Is the `cargo:include` key documented somewhere? I don't see in in the Cargo book
https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script

Reply:
This way of doing in documented here: https://doc.rust-lang.org/cargo/reference/build-script-examples.html?highlight=include#using-another-sys-crate

As far as I know, "include" is not special: any crate with a "links" key can output `cargo:key=value` in its build script, then dependent crates have access to `value` in the environment variable `DEP_[links]_key`.

Reply:
Interesting, so would this cause `DEP_BLIS_INCLUDE` to be defined?

Reply:
Yes. I am using that in my (currently prototype) `blis-sys` crate: https://github.com/cassiersg/blis-sys/blob/master/build.rs

Reply:
Cool, thanks!